### PR TITLE
chores: add chromatic

### PR DESCRIPTION
**Pull request by thiyagaraj-eightfold**
Add chromatic to the project

### CODE REVIEW

**Review by dkilgore-eightfold — APPROVED** [review]

**Review by ychhabra-eightfold — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml
Comment (ychhabra-eightfold) [reply .github/workflows/build.yml:52]: yarn install also needs the correct node version

File: .github/workflows/build.yml
Comment (ychhabra-eightfold) [inline .github/workflows/build.yml:47]: We shouldn't be running this on the pull_request event but only on push event as recommend by the docs see https://www.chromatic.com/docs/github-actions#recommended-configuration-for-build-events

File: .github/workflows/chromatic.yml
Comment (ychhabra-eightfold) [inline .github/workflows/chromatic.yml:9]: No need to create another worklow yaml just add the following in the previous change you had done
```suggestion
    runs-on: ubuntu-latest
    if: github.event_name == 'push'
```

File: .github/workflows/chromatic.yml
Comment (ychhabra-eightfold) [inline .github/workflows/chromatic.yml:18]: actions cache doesnt need node_version

File: .github/workflows/chromatic.yml
Comment (thiyagaraj-eightfold) [reply .github/workflows/chromatic.yml:9]: Want to keep it separate, else it just becomes unmanageable yaml

File: .github/workflows/chromatic.yml
Comment (thiyagaraj-eightfold) [reply .github/workflows/chromatic.yml:18]: its for yarn you mentioned

Comment (ychhabra-eightfold) [reply .github/workflows/chromatic.yml:20]: i meant yarn needs 
- uses: actions/setup-node@v3
   with:
       node-version: ${{ env.NODE_VERSION }}

File: .github/workflows/chromatic.yml
Comment (ychhabra-eightfold) [inline .github/workflows/chromatic.yml:26]: dont think this is needed

**Comment by ychhabra-eightfold** [comment]
the workflow was not correct, you shouldnt merge without approvals @thiyagaraj-eightfold

**Pull request merged**
